src/Core/src/PropertyMapper.cs

```diff
@@ -1,33 +1,58 @@
-#nullable enable
 using System;
-using System.Collections;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace Microsoft.Maui
 {
-	public class PropertyMapper
+	public abstract class PropertyMapper
```

Comment:
This mess is just a git fail because I moved the "chaining" support from the generic `PropertyMapper<T, T>` down to the base mapper. This allows chaining to be done and processed even when there are no views involved.